contrib/completion/zsh/_docker

```diff
@@ -1845,6 +1845,7 @@ __docker_service_subcommand() {
             _arguments $(__docker_arguments) \
                 $opts_help \
                 "($help)*"{-f=,--filter=}"[Filter output based on conditions provided]:filter:->filter-options" \
+                "($help)--format=[Pretty-print services using a Go template]:template: " \
```

Comment:
Yes that's a zsh completion syntax thing. Similar options must be set in the following format `{option1,option2,...}`.
The `=` sign means the value can be set like that: `--option=value` or `--option value`.